package.json

```diff
@@ -155,6 +155,7 @@
         "@storybook/theming": "^8.2.4",
         "@testing-library/jest-dom": "^6.4.6",
         "@testing-library/react": "^16.0.0",
+        "@testing-library/user-event": "^14.5.2",
```

Comment:
do we have to install this? have you tried calling the function without installing it?

Reply:
it's not technically necessary to install since it's a dependency of `@storybook/test` but i think it's probably better to explicitly list it in the package.json anyway. otherwise if storybook/test removed it as a dependency or if we remove storybook entirely then the current tests would break